drivers/wifi/nrf700x/osal/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -223,4 +223,54 @@ struct nrf_wifi_rf_get_xo_value {
 
 #endif /* CONFIG_NRF700X_RADIO_TEST */
 
+/**
+ * @brief This structure defines the parameters used to control the max transmit (TX) power
+ * in both frequency bands for different data rates.
+ *
+ */
+
+struct nrf_wifi_tx_pwr_ceil_params {
+	/*! Maximum power permitted while transmitting DSSS rates in 2.4G band
```

Comment:
This is not the NCS format, please use `/** Member description */`

Reply:
Taken care.